src/examples/echo/echo.go

```diff
@@ -10,12 +10,11 @@ import (
 // change these to test a different UART or pins if available
```

Comment:
this comment doesn't make much sense after the change

Reply:
Should I just remove it entirely?

Reply:
I also think it would be better to delete the comments.

Reply:
OK, done. Thanks for the feedback!